src/components/content/order/retryDeployment/RetryDeployServiceStatusAlert.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Alert } from 'antd';
+import React from 'react';
+import submitAlertStyles from '../../../../styles/submit-alert.module.css';
+import { ApiError, DeployedService, DeployedServiceDetails, Response } from '../../../../xpanse-api/generated';
+import { ContactDetailsShowType } from '../../common/ocl/ContactDetailsShowType';
+import { ContactDetailsText } from '../../common/ocl/ContactDetailsText';
+import useGetOrderableServiceDetailsQuery from '../../deployedServices/myServices/query/useGetOrderableServiceDetailsQuery';
+import OrderSubmitResultDetails from '../orderStatus/OrderSubmitResultDetails';
+
+function RetryDeployServiceStatusAlert({
```

Comment:
i dont think we need a new alert to be defined completely. Only a new error message ust be displayed in the same alert if the 'retry' PUT request failed. 

Also when we click on retry -> all the alerts must behave similar to a fresh order. Currently when i click on retry request, i see request in backend but on the UI there is no change.